README.md

```diff
@@ -166,15 +163,15 @@ If you want to automatically format your code on every commit, you can use [pre-
 Just install it via `pip install pre-commit` and execute `pre-commit install` in the root folder.
```

Comment:
this would stay `pip install`?

Reply:
yes, because it's not related to the repo itself